js/packages/driver/src/helper-android.ts

```diff
@@ -11,23 +13,39 @@ export class HelperAndroid<TDriver, TContext, TElement, TSelector> {
   }): Promise<HelperAndroid<TDriver, TContext, TElement, TSelector> | null> {
     const {spec, driver, logger} = options
     let legacy = false
+    let version = '0.0.0'
     let input = await driver.element({type: 'xpath', selector: '//*[@content-desc="EyesAppiumHelperEDT"]'})
+    const versionElement = await driver.element({
+      type: 'xpath',
+      selector: '//*[@content-desc="EyesAppiumHelper_Version"]',
+    })
     if (!input) {
       legacy = true
       input = await driver.element({type: 'xpath', selector: '//*[@content-desc="EyesAppiumHelper"]'})
     }
+    if (versionElement) {
+      version = await versionElement.getText()
+    }
```

Comment:
I believe this logic has to be much more compact, as well as not spreaded
```suggestion
    let input = await driver.element({type: 'xpath', selector: '//*[@content-desc="EyesAppiumHelperEDT"]'})
    if (!input) {
      legacy = true
      input = await driver.element({type: 'xpath', selector: '//*[@content-desc="EyesAppiumHelper"]'})
    }
    const versionElement = await driver.element({
      type: 'xpath',
      selector: '//*[@content-desc="EyesAppiumHelper_Version"]',
    })
    const version = await versionElement?.getText() ?? '0.0.0'
```